lib/minitest/retry.rb

```diff
@@ -66,7 +75,12 @@ def run_one_method(klass, method_name)
             end
 
             result = super(klass, method_name)
-            break if result.failures.empty?
+
+            if result.failures.empty?
+              break
+            elsif Minitest::Retry.consistent_failure_callback && count == (Minitest::Retry.retry_count - 1)
```

Comment:
Yeah that sounds good. I didn't realize that I hav access to `result` outside of the loop. I see that now.

Update coming.